cmd/util.go

```diff
@@ -22,44 +24,53 @@ func debugPrint() {
 	}
 }
 
+func isInNamespace(meta metav1.ObjectMeta) (valid bool) {
```

Comment:
Could we generalize this function so that it takes a namespace as an argument? Because from reading the function name I expected it to take a namespace as an argument.

And then maybe have a second function `isInRootConfigNamespace` (not the best name, but I guess you get the idea) that calls just `isInNamespace(rootConfig.namespace)`

TLDR & easy solution: rename the function :)

Reply:
I have no better proposal for the moment so I'll take yours.